R/modify_columns.R

```diff
@@ -839,14 +840,13 @@ cols_hide <- function(data,
 cols_merge_uncert <- function(data,
                               col_val,
                               col_uncert,
+                              sep = " +/- ",
```

Comment:
Allow `I(" +/- ")` to mean "use this literal value"

Reply:
Now allowed and explained in docs.